### PR TITLE
Understanding Facial Recognition Using Local Binary Pattern Histogram (LBPH) Algorithm

### LINKED ISSUE #2640: Understanding facial recognition using local binary pattern histogram (LBPH) algorithm
We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner. Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published.

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles to go over.

**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**NOTE: We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

*We're looking for the first 2-3 paragraphs of the article that appropriately summarize what your article will be about.*

### Proposed title of article
Understanding facial recognition using local binary pattern histogram (LBPH) algorithm

### Introduction paragraph (2-3 paragraphs):
This article will explain what LBPH algorithm is, its applications and how is it applied in facial recognition. It will further detail the process of facial recognition using LBPH and gauge its efficiency in performing facial recognition. 
Finally, this article will site some the best application areas of LBPH.

### Key takeaways: 
1. Introduction to facial recognition.
2. The working of LBPH algorithm
3. The two Modes of operation of facial recognition
4. Steps in facial recognition using LBPH
5. Applications of LBPH

### References: 
N/A
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by RuthAMare**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**

Closes #2640.   
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by RuthAMare** [comment]
@collins-a ,

It worked.
PR relating to #2811

**Comment by collins-a** [comment]
> @collins-a ,
> 
> It worked.
> PR relating to #2811

@RuthAMare I'm glad it did. I'll review the changes you made as soon as I can

File: content/articles/understanding-facial-recognition-using-local-binary-histogram-(lbph)-algorithm/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+#### Computational steps
+1.	**The application of the LBP operation:** is the first step of the computational steps. Here, an intermediate image has been created to better represent the original image through a [sliding window](https://www.techopedia.com/definition/869/sliding-window) concept, taking into account two parameters: the `neighbor` and the `radius`. New values are created in the form of binary by comparing the 8 `neighbor` values to the threshold value. For each `neighbor` value greater than the threshold value, the value is set to 1 and 0 for every `neighbor` value less than the threshold value. This forms a matrix of binary numbers excluding the threshold. A central value of the matrix is created by the conversion of the binary number to a decimal value which corresponds to the pixels of the original image. For a better representation of the characteristics of the original image.
+
+The following image illustrates the above conversions:
+[Performing LBP operation on an image](/engineering-education/understanding-facial-recognition-using-local-binary-pattern-histogram-(lbph)-algorithm/lph_operation.png)
```
Comment (collins-a) [inline content/articles/understanding-facial-recognition-using-local-binary-histogram-(lbph)-algorithm/index.md:73]: Just a reminder to add the image source here

File: content/articles/understanding-facial-recognition-using-local-binary-histogram-(lbph)-algorithm/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+2.	**To Extract Histograms:** The image obtained in step is divided into multiple grids, with the help of the Grid parameters `X` and `Y`. This image is in grayscale, each of the histograms of each of the grids is to represent the intensity of the occurrences of each pixel. Each histogram is then combined to create a new histogram that represents the attributes of the original image.
+
+The following image illustrates the above exactraction:
```
Comment (collins-a) [inline content/articles/understanding-facial-recognition-using-local-binary-histogram-(lbph)-algorithm/index.md:77]: Add the image source here as well

**Review by collins-a — COMMENTED** [review]
@RuthAMare good job making the required changes. Just add the image sources to the images included and we're ready for the final review

File: content/articles/understanding-facial-recognition-using-local-binary-histogram-(lbph)-algorithm/index.md (hunk 2 above)
Comment (RuthAMare) [reply content/articles/understanding-facial-recognition-using-local-binary-histogram-(lbph)-algorithm/index.md:79]: > Add the image source here as well

@collins-a, I have done the addition of the image sources.
however i don't know why whenever i push, other commits are picked up as well.
i'm trying to revert if possible

**Comment by RuthAMare** [comment]
@collins-a, I only managed to revert changes on both my local and remote repo back to PRC.
The problem now is that the PR #2857 cannot pick the changes even if i push them, intead it's only opening another PR #2903

**Comment by RuthAMare** [comment]
The PR #2903 has exactly all the changes requested made only that it cannot apply them to #2857

**Comment by collins-a** [comment]
> The PR #2903 has exactly all the changes requested made only that it cannot apply them to #2857

Hello Ruth, apologies for your troubles, let me go over the changes on #2903.

**Pull request closed without merging**